ipn/ipnserver/actor.go

```diff
@@ -237,6 +238,11 @@ func connIsLocalAdmin(logf logger.Logf, ci *ipnauth.ConnIdentity, operatorUID st
 		// Linux.
 		fallthrough
 	case "linux":
+		if !buildfeatures.HasUnixSocketIdentity {
+			// Everybody is an admin if support for unix socket identities
+			// is omitted for the build.
+			return true
```

Comment:
Needn't be part of this change, but do you think we ought to once-log this so it doesn't happen entirely silently? Obviously every time is too much, and maybe it's not worth the extra baggage.

Reply:
Counter argument: we don't do that for FreeBSD or OpenBSD or Illumos? 🤷

Reply:
I'd do it in this change, but there's no proper logger to use.